### PR TITLE
Update redirects in Readme

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

**Comment by n1trux** [comment]
Some of these are wrong, pls fix. Will comment on a few of them.

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@
   * [Archipel](http://archipelproject.org/) - Manage and supervise virtual machines using Libvirt.
   * [CloudStack](http://cloudstack.apache.org/) - Cloud computing software for creating, managing, and deploying infrastructure cloud services.
   * [Cobbler](http://cobbler.github.io/) - Cobbler is a Linux installation server that allows for rapid setup of network installation environments.
-  * [Cracow Cloud One - Polish Private Cloud](http://cc1.ifj.edu.pl/) - The CC1 system provides a complete solution for Private Cloud Computing.
+  * [Cracow Cloud One - Polish Private Cloud](http://cc1.ifj.edu.pl/pl/) - The CC1 system provides a complete solution for Private Cloud Computing.
```
Comment (n1trux) [inline README.md]: this is an english sysadmin list, so I doubt we want to link to a polish version of the site.

Comment (ReadmeCritic) [inline README.md]: @n1trux will fix :sweat_smile:

**Comment by ReadmeCritic** [comment]
@n1trux ready

**Pull request merged**